C/syntax/SyntaxNodes_Expressions.h

```diff
@@ -743,6 +743,49 @@ class PSY_C_API VAArgumentExpressionSyntax final : public ExpressionSyntax
     AST_CHILD_LST2(expr_, typeName_)
 };
 
+/**
+ * \brief The ExtGNU_ChooseExprSyntax class.
+ *
+ * \code
+ * __builtin_choose_expr( const_expr, expr1, expr2 )
+ * \endcode
+ *
+ * \remark \c const_expr is an integer constant expression.
+ */
```

Comment:
Could you add a note for the corresponding AST in clang/llvm (like the notes that exist in other classes)?

```
* \note Similar to:
* - \c clang::ChooseExpr of LLVM/Clang.
 ```

Reply:
:heavy_check_mark: